config/initializers/application_record_serializing.rb

```diff
@@ -10,7 +10,7 @@ def as_json(arg = {})
       attrs[name] = attrs[name].iso8601 if attrs[name].kind_of?(Time)
       attrs[name] = attrs[name].to_s if name.ends_with?("_id") || name == "id"
     end
-    attrs
+    attrs.compact
```

Comment:
Looks like it fails because the some optional values are nil and that doesn't match the schema.  Does swagger really not have a way of defining optional values?